charts/access/slack/README.md

```diff
@@ -0,0 +1,182 @@
+```
+
+### Installing the plugin
+
+```
+helm repo add teleport https://charts.teleport.sh/
+```
+
+```shell
+helm install teleport-plugin-slack teleport/teleport-plugin-slack --values teleport-plugin-slack-values.yaml
+```
+
+Example `teleport-plugin-slack-values.yaml` for using MailGun:
+
+```yaml
+teleport:
+  address: teleport.example.com:443
+  identitySecretName: teleport-plugin-slack-identity
+
+slack:
+  token: xoxb-testtoken
+
+role_to_recipients:
+  '*': access-requests@example.com
+```
+
+Alternatively, you can pass arguments from the command line (useful for one-liners or scripts):
+
+```
+helm install teleport-plugin-slack teleport/teleport-plugin-slack \
+  --set 'teleport.address=teleport.example.com:443' \
+  --set 'teleport.identitySecretName=teleport-plugin-slack-identity' \
+  --set 'slack.token=xoxb-testtoken' \
+  --set 'roleToRecipients.*=access-requests@example.com'
+```
+
+See [Settings](#settings) for more details.
+
+
+## Settings
+
+The following values can be set for the Helm chart:
+
+<table>
+  <tr>
+    <th>Name</th>
+    <th>Description</th>
+    <th>Type</th>
+    <th>Default</th>
+    <th>Required</th>
+  </tr>
+
+  <tr>
+    <td><code>teleport.address</code></td>
+    <td>Host/port combination of the teleport auth server</td>
+    <td>string</td>
+    <td><code>""</code></td>
+    <td>yes</td>
+  </tr>
+  <tr>
+    <td><code>teleport.identitySecretName</code></td>
+    <td>Name of the Kubernetes secret that contains the credentials for the connection</td>
+    <td>string</td>
+    <td><code>""</code></td>
+    <td>yes</td>
+  </tr>
+  <tr>
+    <td><code>teleport.identitySecretPath</code></td>
+    <td>Key of the field in the secret specified by <code>teleport.identitySecretName</code></td>
+    <td>string</td>
+    <td><code>"auth_id"</code></td>
+    <td>yes</td>
+  </tr>
+
+  <tr>
+    <td><code>slack.token</code></td>
+    <td>Slack API token</td>
+    <td>string</td>
+    <td><code>""</code></td>
+    <td>yes</td>
+  </tr>
+
+  <tr>
+    <td><code>roleToRecipients</code></td>
+    <td>
+      Mapping of roles to a list of channels and Slack emails. <br />
+      Example:
+      <pre>
+"dev" = ["dev-access-requests", "user@example.com"]
+"*" = ["access-requests"]</pre>
+    </td>
+    <td>map</td>
+    <td><code>{}</code></td>
+    <td>yes</td>
+  </tr>
+
+  <tr>
+    <td><code>log.output</code></td>
+    <td>
+      Logger output. Could be <code>"stdout"</code>, <code>"stderr"</code> or a file name,
+      eg. <code>"/var/lib/teleport/slack.log"</code>
+    </td>
+    <td>string</td>
+    <td><code>"stdout"</code></td>
+    <td>no</td>
+  </tr>
+  <tr>
+    <td><code>log.severity</code></td>
+    <td>
+      Logger severity. Possible values are <code>"INFO"</code>, <code>"ERROR"</code>,
+      <code>"DEBUG"</code> or <code>"WARN"</code>.
+    </td>
+    <td>string</td>
+    <td><code>"INFO"</code></td>
+    <td>no</td>
+  </tr>
+</table>
```

Comment:
We'd usually represent tables in Markdown using table syntax: https://github.github.com/gfm/#tables-extension-

Was there a reason why you preferred HTML syntax here?

Reply:
The main reason is that the table format doesn't support code blocks (`  ``` `) for multi-line examples, unfortunately...